stack.yaml

```diff
@@ -1,15 +1,15 @@
 # SPDX-FileCopyrightText: 2020 TQ Tezos
 # SPDX-License-Identifier: LicenseRef-MIT-TQ
 
-resolver: lts-16.11
+resolver: lts-16.5
```

Comment:
@gromakovsky I've changed this due to `morley` and `morley-ledgers` both have `resolver: lts-16.5`.

Reply:
Yep, I assumed the same. I don't mind.